modules/earthengine-layers/src/tile-layer/tile-2d-header.js

```diff
@@ -0,0 +1,72 @@
+import {log} from '@deck.gl/core';
```

Comment:
Do we need all these files? This is not supported by deck.gl 8.2?

Reply:
You're right it can be deleted. I think I had to copy it for the initial TerrainLayer implementation and forgot to remove it.